index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body class="all">
+    <header>
+        <nav>
+            <img class="navbar" src="./images/Bars.svg" alt="bars"> 
+        </nav>
+    </header>
+    <section>
+        <div class="introduction">
+            <div class="texts">
+                <h1 class="welcome">Hey there. I'm Assem.</h1>
+                  <h1 class="green">I'm a software developer</h1>
```

Comment:
Greeting!

You did a great work so far, but there are still some changes you'll need to perform before going to the next step. 😄 

[OPTIONAL] I suggest you use a `span` tag inside the `h1` to make you code more readable and respecting the DRY method, you can do it like that:

```
<h1 class="welcome">
  Hey there.
  <br />
  I'm Assem.
  <br />
  <span class="green">I'm a software developer</span>
</h1>
```

According to the Figma template, I suggest you stick to the Design as much as possible following `font, margin/padding, colors, ect.`. 😄 

> ### Your Desgin
>
><img width="375" alt="Screenshot 2023-07-03 at 3 15 32 PM" src="https://github.com/zhorabay/My-portfolio/assets/91237525/5f698ddf-12d3-408b-99ec-dd40c57122d3">
>
> ## The Figma Design
>
><img width="375" alt="Screenshot 2023-07-03 at 2 57 16 PM" src="https://github.com/zhorabay/My-portfolio/assets/91237525/7cf80592-aad5-436d-b7c0-ada8c93aa97a">